doc/API.md

```diff
@@ -68,27 +69,37 @@ The currently-defined error responses are:
 * status code 400, errno 101: Invalid email. Missing or malformed email.
 * status code 400, errno 102: Invalid password. The password should have a minimum of 8 chars.
 * status code 400, errno 103: Missing or malformed authentication header.
+* status code 400, errno 104: Invalid user id.
 * status code 401, errno 401: Unauthorized. If credentials are not valid.
 * status code 409, errno 409: Conflict. The user is already registered.
 * status code 410, errno 410: Gone. The resource is no more available. Don't insist.
+* status code 423, errno 423: Locked. You are trying to delete the last user with admin privileges. That's forbidden.
 * status code 501, errno 501: Internal server error.
 * any status code, errno 999: Unknown error
 
 # API Endpoints
 
 * Setup
     * [POST /setup](#post-setup)
-    * [POST /login](#post-login)
+* Login
+    * [POST /login](#post-login) :lock: (CORS allowed)
+* User management
+    * [POST /users](#post-users) :lock:
```

Comment:
So we are forcing the creating of users in the box, not allowing cors for this methods right? It's ok for me, sounds like a good decision.